ui/lib/bulma/index.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-env node */
+/* eslint node/no-extraneous-require: 0 */
```

Comment:
What's the extraneous require here?

Reply:
hmm yeah… from [here](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-extraneous-require.md):

> If a `require()`'s target is extraneous (it's not written in `package.json`), the program works in local, but will not work after dependencies are re-installed. It will cause troubles to your team/contributors. This rule disallows `require()` of extraneous modules.

These are the linting errors that happen without it:

```
   5:22  error  "broccoli-funnel" is extraneous  node/no-extraneous-require
  22:51  error  "bulma" is extraneous            node/no-extraneous-require
```

The `bulma` one doesn’t make sense to me because that’s in `package.json`. `broccoli-funnel` isn’t but is present in `yarn.lock` because it’s required by various dependencies. I didn’t think too hard about this and just added the ESLint override rather than spend any time trying to fix it 😬 but if you’d prefer to not have this hackish override, I can try to figure it out, let me know!

Reply:
Ah, so not strictly `require`, just the act of requiring, even if via `import. Makes sense.

Bulma, btw, is not in the package.json in this case. This is an in-repo addon and gets its own package.json:

https://github.com/hashicorp/nomad/blob/1064b9f71f65ec7116e4336d45ed1d1192ea560d/ui/lib/bulma/package.json#L1-L6

Reply:
oh nice, I wasn’t aware of that. I’ve added the dependencies to that file in 2ae6a5e and removed this ESLint override, thanks!